src/util/helpers.js

```diff
@@ -136,3 +136,6 @@ export const getStartEndDateError = (startDate, endDate) => {
     }
     return null;
 };
+
+// Remove line breaks from text (not displayed corrently in map downloads)
+export const removeLineBreaks = text => text.replace(/(\r\n\t|\n|\r\t)/gm, '');
```

Comment:
We are we matching tabs here?

Reply:
Changed to /\r?\n|\r/g and added a comment about the source :-)